docs/reference.rst

```diff
@@ -165,6 +165,13 @@ A valid manifest file must have the file extension ``.raucm``.
   List of hooks enabled for this bundle.
 
 
+**[handler] section**
+
+``filename``
+  Handler script path name, relative to the bundle content. Used to fully
+  replace default update process.
```

Comment:
The `[handler]` section also takes an `args` argument which should be described here, too.

What about adding at least a single line:

    ``args``
    Arguments to pass to the handler script, such as ``args=--verbose``